### PR TITLE
add SubhamRaoniar28

### LINKED ISSUE #397: Subham Raoniar
Subham-Raoniar-github-profile

### PR DESCRIPTION
**Pull request by SubhamRaoniar28**
resolve #397

### CODE REVIEW

**Comment by SubhamRaoniar28** [comment]
Please merge my changes.

File: data.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 [
+  {
```
Comment (AllanRegush) [inline data.json:2]: Please add yourself to the bottom.

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1:

File: data.json (hunk 1 above)
Comment (SubhamRaoniar28) [reply data.json]: Alright, done

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you :smile:

File: data.json (hunk 1 above)
Comment (SubhamRaoniar28) [reply data.json]: Thanks

**Comment by SubhamRaoniar28** [comment]
Thanks 😃

**Pull request merged**